Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ruby:2.3.1
```

Comment:
Looks like this project runs on [ruby 2.3.0](https://github.com/alphagov/collections-publisher/blob/master/.ruby-version)